### PR TITLE
Fixed Typo's, Moved Owners Array to config.json and Added More Variety In Meme Subreddits

### PR DESCRIPTION
**Pull request by dotslashh**
- Typo Fixed in InfoCommand.ts
- Added r/memes and r/dankmemes to an array of subreddits, which is chosen at random, on command execution.
- Moved Owners Array (ID's) to the config.json for easier configuration.
- Added config-example.json to show what a config.json should look like.

### CODE REVIEW

File: src/commands/Fun Commands/MemeCommand.ts [hunk 1]
```diff
@@ -3,7 +3,8 @@ import { Util } from 'discord.js';
 import * as ImageAPI from 'imageapi.js';
 
 export const run: RunFunction = async (client, message) => {
-	const meme = await ImageAPI.advanced('meme');
+	let memeSubreddits = ['meme', 'memes', 'dankmemes']
```
Comment (Milo123459) [inline src/commands/Fun Commands/MemeCommand.ts:6]: use const over let

**Comment by dotslashh** [comment]
Any reason as to why the config-example was removed?

**Review by Milo123459 — COMMENTED** [review]
> Any reason as to why the config-example was removed?

don't see a point really

File: src/commands/Fun Commands/MemeCommand.ts (hunk 1 above)
Comment (dotslashh) [reply src/commands/Fun Commands/MemeCommand.ts:6]: This doesn't need to be changed, as the array is never going to be modified.

**Comment by dotslashh** [comment]
> > Any reason as to why the config-example was removed?
> 
> don't see a point really

It will help people host the bot if they need to, at least give them a guide on what config they need.

**Review by Milo123459 — COMMENTED** [review]
> > > Any reason as to why the config-example was removed?
> > 
> > 
> > don't see a point really
> 
> It will help people host the bot if they need to, at least give them a guide on what config they need.

If they're going to be self hosting it I expect them to be competent enough to read an interface and generate a config.json based on that, so yea, I don't see a point imo

**Review by Milo123459 — APPROVED** [review]
all good

**Comment by dotslashh** [comment]
its a bit of a faff apparently

**Pull request merged**